pkg/bot/slack_cloud.go

```diff
@@ -111,7 +111,6 @@ func NewCloudSlack(log logrus.FieldLogger,
 		clusterName:      clusterName,
 		realNamesForID:   map[string]string{},
 		msgStatusTracker: NewSlackMessageStatusTracker(log, client),
-		messages:         make(chan *pb.ConnectResponse, platformMessageChannelSize),
 		messageWorkers:   pool.New().WithMaxGoroutines(platformMessageWorkersCount),
```

Comment:
tested e2e, unfortunately also got panic:
```
panic: send on closed channel

goroutine 171 [running]:
github.com/sourcegraph/conc/pool.(*Pool).Go(0xc00098c280, 0xc0009ef680)
	/Users/mszostok/workspace/go/pkg/mod/github.com/sourcegraph/conc@v0.3.0/pool/pool.go:63 +0xe7
github.com/kubeshop/botkube/pkg/bot.(*CloudSlack).startMessageProcessor(0xc000a92480, {0x39791b8?, 0xc0009ec7d0})
	/Users/mszostok/workspace/go/src/github.com/kubeshop/botkube/pkg/bot/slack_cloud.go:228 +0x130
created by github.com/kubeshop/botkube/pkg/bot.(*CloudSlack).start in goroutine 99
	/Users/mszostok/workspace/go/src/github.com/kubeshop/botkube/pkg/bot/slack_cloud.go:203 +0x61e
```

we shutdown worker service, we need to also recreated it.

Reply:
Thank you, fixed here: https://github.com/kubeshop/botkube/pull/1264/commits/1d4becd23492bfd28c6fc67c0bb44ad17e851ca7